packages/server/graphql/mutations/updateTaskDueDate.ts

```diff
@@ -24,9 +25,8 @@ export default {
   },
   async resolve(
     _source: unknown,
-    // FIXME type mismatch, dueDate is Date | undefined
-    {taskId, dueDate}, //: {taskId: string; dueDate?: Date | null},
-    {authToken, dataLoader, socketId: mutatorId}
+    {taskId, dueDate}: {taskId: string; dueDate?: Date},
```

Comment:
+1 optional GraphQL types should be `dueDate: Date | undefined | null` since `null` will be passed when omitted